contrib/kube-prometheus/assets/prometheus/rules/prometheus.rules.yaml

```diff
@@ -90,3 +90,12 @@ groups:
     annotations:
       description: "Prometheus {{ $labels.namespace }}/{{ $labels.pod}} isn't ingesting samples."
       summary: "Prometheus isn't ingesting samples"
+
+  - alert: PrometheusTargetScapesDuplicate
+    expr: increase(prometheus_target_scrapes_sample_duplicate_timestamp_total[2h]) > 0
+    for: 4h
```

Comment:
This is a pretty long time range, `10m` would over a `[5m]` range query would give the same result but you'd be alerting much quicker.

Reply:
Yep nice